static/fontscale.js

```diff
@@ -80,6 +80,8 @@ function FontScale(domRoot, state, fontSelectorOrEditor) {
     this.domRoot = domRoot;
     // Old scale went from 0.3 to 3. New one uses 8 up to 30, so we can convert the old ones to the new format
     this.scale = state.fontScale || options.defaultFontScale;
+    // The check seems pointless, but it ensures a false in case it's undefined
```

Comment:
Perfect! Love it. Thanks for thinking of , and handling, this case!

Should we default the sizes etc to use px now and set this to true in the "Reset UI" do you think? I'm not against having everything unified size now. Not necessarily in this PR!

Reply:
I think that's already the behaviour with this line, right? https://github.com/compiler-explorer/compiler-explorer/pull/2521/files#diff-0301fe1aacc8e8f15f982b21f556e93d0dd812dc0bc468ac3868d6a5bcc9d86fR107